doc/manual/interfaces.rst

```diff
@@ -245,7 +245,6 @@ As a more complicated example, let's define our own toy N-dimensional sparse-lik
            end
            SparseArray{T}(::Type{T}, dims::Int...) = SparseArray(T, dims)
            SparseArray{T,N}(::Type{T}, dims::NTuple{N,Int}) = SparseArray{T,N}(Dict{NTuple{N,Int}, T}(), dims)
-    SparseArray{T,N}
```

Comment:
this is the output of the type definition, deleting it would make the doctest fail